collector/kubernetes/Makefile

```diff
@@ -0,0 +1,21 @@
+
+.PHONY: run-cluster
+run-cluster:
+	kind create cluster --name kubernetes
```

Comment:
If we could change the cluster name ... something like `k8s_demo` I think is more descriptive.